code/stat159lambda/classification/partition_volumes.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+from stat159lambda.config import REPO_HOME_PATH
+
+NUM_OFFSET_VOLUMES = 9
+NUM_VOLUMES = 3543
```

Comment:
may want to change this later to make it more generalizable to other projects but this is okay for now